baby-bear/src/baby_bear.rs

```diff
@@ -421,6 +439,13 @@ fn monty_reduce(x: u64) -> u32 {
     x_sub_u_hi.wrapping_add(corr)
 }
 
+#[inline]
+pub fn sum_u64(vec: &[BabyBear]) -> BabyBear {
+    BabyBear {
+        value: (vec.iter().map(|x| (x.value as u64)).sum::<u64>() % (P as u64)) as u32,
+    }
+}
+
```

Comment:
The only reason I didn't do this is that this function is slower for small summations (E.g. length < 8). More than happy to make this change but it might be worth noting that somewhere.

Reply:
I'm going to pull this summation stuff out of the current PR and move it into a separate PR as it's essentially independent.